src/Bridge/Elasticsearch/DataProvider/Filter/OrderFilter.php

```diff
@@ -70,11 +70,11 @@ public function apply(array $clauseBody, string $resourceClass, ?string $operati
             $order = ['order' => $direction];
 
             if (null !== $nestedPath = $this->getNestedFieldPath($resourceClass, $property)) {
-                $nestedPath = null === $this->nameConverter ? $nestedPath : $this->nameConverter->normalize($nestedPath, $resourceClass, null, $context);
```

Comment:
can't you use the CamelCaseToSnakeCaseNameConverter directly ? Or just provide a new NameConverter that does the same thing as you added in snakeCasePropertyPath ?

Reply:
`snakeCasePropertyPath` uses the `CamelCaseToSnakeCaseNameConverter` but also takes care of nested paths like `someCamelCaseParent.someCamelCaseChild`, which also needs to be handled. And since this needs to be done in all search filters I thought adding it to the abstract filter was the way to go.

As for providing a NameConverter, where can we do that only on entity level? And only on the ES query (keep camelCasing in input/output)?

Reply:
see my other comment which is more appropriate for what I meant